OTKit/README.md

```diff
@@ -19,15 +19,41 @@
 $ npm install --save-dev <token-name>
 ```
 
+### Install postcss-pxtorem for Pixel -> Rem conversion
+All contributions to this project should be in pixels.  For users of these tokens, you will have to use [postcss-pxtorem](https://github.com/cuth/postcss-pxtorem) to handle the pixel to rem conversion in your CSS files.
+
+```
+npm install --save-dev postcss-pxtorem
+```
+
+Then add it to your [postcss](https://github.com/postcss/postcss) processors with the following recommend configuration:
+
+```
+import pxtorem from 'postcss-pxtorem';
+
+const processors = [
+  pxtorem({
+    rootValue: 16,
```

Comment:
While this is the default value for pxToRem, this is just a critical thing, that I didn't want to risk pxToRem changing the default one day for whatever reason.

Reply:
can't we get this value directly from the design token itself?

Reply:
Yup, though with one caveat.  People will not be able to do this if they use the JSON configuration approach for postcss.  However, i think that will be relatively uncommon for most projects at OpenTable.

Reply:
Feel free to do this in this PR or in a follow up